utils/frame/benchmarking-cli/src/template.hbs

```diff
@@ -5,14 +5,15 @@
 //! DATE: {{date}}, STEPS: `{{cmd.steps}}`, REPEAT: {{cmd.repeat}}, LOW RANGE: `{{cmd.lowest_range_values}}`, HIGH RANGE: `{{cmd.highest_range_values}}`
 //! EXECUTION: {{cmd.execution}}, WASM-EXECUTION: {{cmd.wasm_execution}}, CHAIN: {{cmd.chain}}, DB CACHE: {{cmd.db_cache}}
 
-// Executed Command:
+/* Executed Command:
 {{#each args as |arg|}}
-// {{arg}}
-{{/each}}
+    {{arg}}
+{{/each}} */
 
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(clippy::unnecessary_cast)]
```

Comment:
we probably have lots of clippy warnings all over the repo because we don't enforce it. I am happy to discuss adding it, but while that's not the case, these manual `allow(clippy::xxx)` are just noise in the code IMO.

Reply:
I think people shouldn't run clippy on our code, but should be able to run clippy on their code without too much issue.
So macro generate code and template can be fixed by contributor if they want IMO

Reply:
fair point 👍